msteams-platform/bots/how-to/authentication/add-authentication.md

```diff
@@ -280,9 +280,9 @@ With the preliminary settings done, let's focus on the creation of the bot to us
 
 # [JavaScript](#tab/node-js)
 
-1. Clone [node-auth-sample][teams-auth-bot-js].
+1. Clone [node-auth-sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-teams-authentication/nodejs).
```

Comment:
Please don't directly add the link here:

There will be a section for `[teams-auth-bot-js]` so please add the link there or please create another section if duplicate.